phpspec.yml.ci

```diff
@@ -0,0 +1,10 @@
+suites:
+  data_import_suite:
+    namespace: Ddeboer\DataImport
+    psr4_prefix: Ddeboer\DataImport
+extensions:
+  - PhpSpec\Extension\CodeCoverageExtension
+formatter.name: pretty
+code_coverage:
```

Comment:
> Behat can be very useful for client-focused web tests

That is true, but what you think is the Mink extension. Behat is a StoryBDD test tool.

Behat is absolutely fine for integration tests. The point of integration tests is that you test your library with some real input and see what would happen in a real environment. With behat's "If I do this then this should happen" workflow this is absolutely possible.

> they should make sense to developers

I am not sure what that means, but

```
When I pass in an empty string
I should see <outcome>
```

is definitely more readable than

`testWithEmptyString`

I admit it is harder to maintain at first, but as you get used to it, it gets easier.

However I don't have a problem with PHPUnit either.